hangupsbot/commands.py

```diff
@@ -304,3 +305,36 @@ def roll(bot, event, *args):
             i += 1
         message = 'Rolling {}: {} ({})'.format(arg, results, total)
         bot.parse_and_send_segments(event.conv, message)
+
+
+def get_topic(seed):
```

Comment:
Would also be nice to somehow build each command in a separate file that is automatically included somehow.  The commands.py file will get large and disorganized fast.